.github/workflows/build-and-publish.yml

```
@@ -37,10 +44,16 @@ jobs:
         env:
           CIBW_SKIP: "cp36-*" # skip 3.6
           CIBW_ARCHS_MACOS: "x86_64 universal2 arm64"
+          CIBW_ARCHS_LINUX: "auto aarch64"
       - uses: actions/upload-artifact@v4
         with:
           name: python-package-distributions-macos
           path: ./wheelhouse/*.whl
+      - uses: actions/upload-artifact@v4
+        with:
+          name: python-package-distributions-linux
+          path: ./wheelhouse/*.whl
```

I guess this could be simplified to:

```suggestion
          name: python-package-distributions-wheels
          path: ./wheelhouse/*.whl
```

---

yes, that will work too. Actually I did it differently when testing it in my repo: https://github.com/vermut/marisa-trie/commit/1598f50c00971255d9a386d4fcd10dbabdbb7674

also note setuptools addition - I had issue with that in on the runs: https://github.com/vermut/marisa-trie/actions/runs/8961082399/job/24608308956

---

Yup, I just saw the `setuptool` issue. I'll look into it right now to be able to cut a new release.

---

Your version is way better, could you apply it here too?

---

Ah, and you already fixed the `setuptools` issue. Let's also fix it in that PR :)

---

I'll just create a PR from my master. Squash the WIP commits when merging.